src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,35 @@
+body,
+ul {
+  margin: 0;
+}
+
+.nav {
+  &__list {
+    padding: 0;
```

Comment:
Why navigation is in `reset.scss` file?
Create a separate file for each BEM block